Makefile

```diff
@@ -36,13 +42,13 @@ build:
 	rm -f build-cfn/*
 
 # create build artifacts (= zip files) for CloudFormation deployment
-	git archive --format zip --output build-cfn/sandbox-accounts-for-events.zip main
+	git archive --format zip --output build-cfn/sandbox-accounts-for-events.zip $(branch)
 	cd install/cfn-lambda/dceHandleTerraFormDeployment && zip -r ../../../build-cfn/sandbox-accounts-for-events-lambda-terraform.zip . && cd -
 	cd install/cfn-lambda/dceHandleAmplifyDeployment && zip -r ../../../build-cfn/sandbox-accounts-for-events-lambda-amplify.zip . && cd -
 
 # upload build artifacts and CloudFormation template to specified S3 bucket
 	aws s3 sync build-cfn s3://$(bucket) $(profileString)
-	aws s3 cp install/sandbox-accounts-for-events-install.yaml s3://$(bucket)/sandbox-accounts-for-events-install.yaml $(profileString)
+# aws s3 cp install/sandbox-accounts-for-events-install.yaml s3://$(bucket)/sandbox-accounts-for-events-install.yaml $(profileString)
```

Comment:
@mavogel Why did you comment out the line copying the CFN template to the S3 bucket? Is this in purpose or a test leftover?

Reply:
it is a test leftover. Thank you for pointing out! I am still working on the fixes and will make the PR then ready

Reply:
I will revert this line to be able to merge the changes